### PR TITLE
port one clothing item from girlystation

### PR DESCRIPTION
**Pull request by optimumtact**
## Changelog
:cl: oranges
add: Added human wearable pet collars
/:cl:

for all those petplay peeps out there, hit me up

### CODE REVIEW

**Comment by ArcaneMusic** [comment]
sin.

**Comment by Iamgoofball** [comment]
Looks good to me. Ship it.

**Comment by SweptWasTaken** [comment]
jaw drops

eyes pop out of head

tongue rolls out

HUMINA HUMINA HUMINA!

AWOOGA AWOOGA!

EE-AW EE-AW!

BOIOIOING!

picks up jaw

fixes eyes

rolls up tongue

buries face in ass

BLBLBLBLBL LBLBLBLBLBLBLLB

WHOA MAMA

**Comment by tralezab** [comment]
coil

File: code/modules/clothing/neck/_neck.dm [hunk 1]
```diff
@@ -267,3 +267,26 @@
...
+	icon_state = "petcollar"
+	item_color = "petcollar"
+	locked_string = "is stuck."
+/obj/item/clothing/neck/petcollar_human/equipped(mob/user, slot)
+	if(slot==SLOT_NECK)
+		ADD_TRAIT(src,TRAIT_NODROP,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_MONKEYLIKE,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_DUMB,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_PACIFISM,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_TRAPPED,CLOTHING_TRAIT)
```
Comment (Couls) [inline code/modules/clothing/neck/_neck.dm]: We don't actually have trait_trapped and i'm scared to ask what it does on girlystation

**Comment by Tharcoonvagh** [comment]
Headcoder shitposts
Cringe

**Comment by tralezab** [comment]
issuemanager

**Comment by Twaticus** [comment]
Hi! This PR seems to be missing headpats. I will not be able to merge this in an unfinished state, please fix it.

**Comment by ArcaneMusic** [comment]
KirbyEmote2

**Comment by tralezab** [comment]
pithing

**Comment by tralezab** [comment]
buzzy_bee_tip

**Comment by tralezab** [comment]
if oranges scrolls down this far he has to merge https://github.com/tgstation/tgstation/pull/49062

**Comment by trollbreeder** [comment]
Oranges is drunk again. Time to post my trump cards.
#37534
https://en.m.wikipedia.org/wiki/Cummingtonite

**Comment by Tharcoonvagh** [comment]
do-you-think-god-stays-in-heaven-because-he-too-lives-in-fear-of-what-he-amp-039-s-created_o_6994781

**Comment by trollbreeder** [comment]
on a more serious note, make this only wearable by nonhumans to show our pride and to show that these filthy lizards are slaves to our kind

**Comment by tralezab** [comment]
GOT EM
image

File: code/modules/clothing/neck/_neck.dm (hunk 1 above)
Comment (trollbreeder) [reply code/modules/clothing/neck/_neck.dm]: he probably means the handcuffed trait instead

**Review by spookydonut — DISMISSED** [review]

**Comment by trollbreeder** [comment]
add the against god and nature tag and the fuck tag please

File: code/modules/clothing/neck/_neck.dm (hunk 1 above)
Comment (trollbreeder) [reply code/modules/clothing/neck/_neck.dm]: #49501

File: code/modules/clothing/neck/_neck.dm [hunk 2]
```diff
@@ -267,3 +267,26 @@
...
+
+/obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
+	if(ishuman(M))
+		return FALSE
+	return ..()
+/obj/item/clothing/neck/petcollar_human
+	name = "pet collar (human)"
+	desc = "It's for people who act like pets."
+	icon_state = "petcollar"
+	item_color = "petcollar"
```
Comment (spookydonut) [inline code/modules/clothing/neck/_neck.dm]:
```suggestion
```

File: code/modules/clothing/neck/_neck.dm [hunk 3]
```diff
@@ -267,3 +267,26 @@
...
+	locked_string = "is stuck."
```
Comment (spookydonut) [inline code/modules/clothing/neck/_neck.dm]:
```suggestion
```

File: code/modules/clothing/neck/_neck.dm [hunk 4]
```diff
@@ -267,3 +267,26 @@
...
+/obj/item/clothing/neck/petcollar_human/equipped(mob/user, slot)
+	if(slot==SLOT_NECK)
```
Comment (spookydonut) [inline code/modules/clothing/neck/_neck.dm]:
```suggestion
	if(slot==ITEM_SLOT_NECK)
```

File: code/modules/clothing/neck/_neck.dm [hunk 5]
```diff
@@ -267,3 +267,26 @@
...
+		ADD_TRAIT(src,TRAIT_NODROP,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_MONKEYLIKE,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_DUMB,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_PACIFISM,CLOTHING_TRAIT)
+		ADD_TRAIT(user,TRAIT_TRAPPED,CLOTHING_TRAIT)
+		to_chat(user,"<span class='danger'>You feel pretty helpless...</span>")
+
```
Comment (spookydonut) [inline code/modules/clothing/neck/_neck.dm]:
```suggestion
	return ..()

```

**Review by spookydonut — CHANGES_REQUESTED** [review]

**Comment by mrhugo13** [comment]
check your reddit oranges :)

**Comment by mrhugo13** [comment]
doomgers

**Comment by optimumtact** [comment]
GET OFF MY PR, I"LL PUT YOU IN A LOCKED OUTFIT

**Comment by mrhugo13** [comment]
merge it oranges

**Comment by mrhugo13** [comment]
do it

**Comment by Time-Green** [comment]
dont

**Comment by SweptWasTaken** [comment]
LOCKED SCHOOLGIRL OUTFIT IF YOU DOWNVOTE

**Comment by Twaticus** [comment]
Hello! I can see my comment was marked as "out-of-date", but it seems the issue either wasnt fixed or the change wasnt included in the changelog. Please fix the issues I brought up or update the changelog before marking my comment as off-topic, thank you.

**Comment by optimumtact** [comment]
whoopslol

**Comment by Twaticus** [comment]
delete this

**Comment by PKPenguin321** [comment]
tfw no girthystation ports

**Comment by HomocidlePancakes** [comment]
11E5BAF3-5536-4483-831A-B1D7BA80215D
Hmmm... Oranges is adding a clothing item. I wonder what it will be
B0092DE0-359A-4DCE-A07D-8C35D8ECBD2F

Unsurprising.

**Review by spookydonut — APPROVED** [review]

**Comment by Archanial** [comment]
Fetish content, speedmerge.

**Comment by Putnam3145** [comment]
i should've known that there's a global SS13 conservation of horny mechanics

**Comment by Ghommie** [comment]
Numbers of days since oranges crossed the line: 0

**Comment by AnturK** [comment]
Time to take oranges behind the shed.

**Comment by AffectedArc07** [comment]
Put me in the reddit screenshot

**Comment by Archanial** [comment]
redditor

**Comment by Anonmare** [comment]
Based

**Comment by Nickkoranda** [comment]
Dubs check 'em

**Comment by notfrying1pans** [comment]
balls

**Pull request closed without merging**